arches/app/media/js/utils/map-popup-provider.js

```diff
@@ -1,7 +1,9 @@
-define(['arches',
+define([
+    'arches',
     'knockout',
+    'turf',
```

Comment:
This is unused so I think it should be removed.

Reply:
fix in latest